packages/express/README.md

```diff
@@ -38,14 +38,6 @@ This is equivalent to manually setting `$NODE_ENV` before calling the actual com
 $ NODE_ENV=production un serve
 ```
 
-##### `-s` / `--static`
```

Comment:
This one is a bit tricky. It makes sense to remove it, since untool does not do anything with it anymore, but what about other mixins wanting to implement custom rewriting for static mode?

They still could use the `static` flag, right?
Thinking about it: There is currently no way for mixins to add command line arguments to untool "core" commands (build, serve) is it?

Reply:
True, but at least you could add them globally, like `@untool/yargs` used to do in `handleArguments`.

Reply:
@dmbch I don't understand your comment - can you explain / add a link to the code?

Reply:
Sure: https://github.com/untool/untool/blob/0bd5c3a47e7f913e90f94acfbe76ebdc6687e9b1/packages/yargs/mixin.core.js#L11

Reply:
@dmbch ok, so it's global arguments - I don't think that that is a solution we should encourage.
So we would have to extend the mixin class and overwrite the `registerCommands` method.. :/

Reply:
Either that, introduce a new command or rely on config to decide wether to rewrite or not.